CHANGELOG.md

```diff
@@ -99,6 +99,9 @@
 
 #### Bug Fixes
 
+* Adding a nesting rule configuration option:`ignore_nesting_typealias_and_associatedtype` that Typealias and AssociatedType will be ignored from the analysis of the rule.
+  [#3183](https://github.com/realm/SwiftLint/issues/3183)
```

Comment:
```suggestion
* Add new option `ignore_typealiases_and_associatedtypes` to
  `nesting` rule. It excludes `typealias` and `associatedtype`
  declarations from the analysis.  
  [marunomi](https://github.com/marunomi)
  [#3183](https://github.com/realm/SwiftLint/issues/3183)
```

Reply:
Thank you for suggesting the description!